src/bugreporter/odemis_bugreporter.py

```diff
@@ -310,6 +310,8 @@ def compress_files(self):
             outputs["ps.txt"] = self._get_future_output(['/bin/ps', 'aux'], timeout=60)
             outputs["uname.txt"] = self._get_future_output(['/bin/uname', '-a'], timeout=5)
             outputs["ip.txt"] = self._get_future_output(['/bin/ip', 'address'], timeout=30)
+            outputs["memory.txt"] = self._get_future_output(['/usr/bin/free'], timeout=5)
+            outputs["df.txt"] = self._get_future_output(['/bin/df'], timeout=10)
```

Comment:
Maybe also storing the dmesg output could be useful?

Reply:
Yes, we actually do this indirectly, by saving "/var/log/syslog", which contains the dmesg output.

Reply:
Aaah i didn't spot that line, great!